### PR TITLE
fix: invalid method call in interactions walkthrough

### PR DESCRIPTION
**Pull request by lazpavel**
#### Description of changes
— fixed reference to invalid method name from prompter.prompt to prompter.input

#### Description of how you validated changes
— manual test

#### Checklist
- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-interactions/src/provider-utils/awscloudformation/service-walkthroughs/lex-walkthrough.js [hunk 1]
```diff
@@ -386,7 +386,7 @@ async function addSlot(context, intentName, resourceName, parameters) {
       slot.type = slot.type[0];
     }
 
-    slot.prompt = await prompter.prompt('Enter a prompt for your slot (e.g. What city?)', {
+    slot.prompt = await prompter.input('Enter a prompt for your slot (e.g. What city?)', {
```
Comment (sobolk) [inline packages/amplify-category-interactions/src/provider-utils/awscloudformation/service-walkthroughs/lex-walkthrough.js:389]: should we add test for this ?

Comment (sobolk) [reply packages/amplify-category-interactions/src/provider-utils/awscloudformation/service-walkthroughs/lex-walkthrough.js:389]: or convert to TS

Comment (lazpavel) [reply packages/amplify-category-interactions/src/provider-utils/awscloudformation/service-walkthroughs/lex-walkthrough.js:389]: Like the idea of converting to TS, will do, from the initial try this file is a mess :)

**Review by sdstolworthy — DISMISSED** [review]

File: .eslint-dictionary.json [hunk 2]
```diff
@@ -222,6 +222,7 @@
   "kinesis",
   "lang",
   "lerna",
+  "lillies",
```
Comment (sobolk) [inline .eslint-dictionary.json:225]: I recall fixing this. Do we still need it here?

**Review by sobolk — APPROVED** [review]

**Review by sdstolworthy — APPROVED** [review]

**Pull request merged**